package.json

```diff
@@ -1,19 +1,34 @@
 {
-  "name": "developersindia.in",
```

Comment:
Remove unnecessary spam from this file & then we'll get back to setting up pre-commit scripts.